Cubical/Structures/Axiom.agda

```diff
@@ -0,0 +1,43 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Structures.Axiom where
```

Comment:
Wouldn't this read nicer if it was `Axioms`? (I don't feel super strongly about this so feel free to ignore)